go/stellar/stellarsvc/service_test.go

```diff
@@ -33,11 +33,11 @@ func TestCreateWallet(t *testing.T) {
 	_, tcs, cleanup := setupNTests(t, 2)
 	defer cleanup()
 
-	created, err := CreateWallet(context.Background(), tcs[0].G)
+	created, err := stellar.CreateWallet(context.Background(), tcs[0].G)
```

Comment:
Could these tests still use the service handlers when possible? That way they cover the most.

In this particular case WalletInitLocal

Reply:
I started to do that, but the service handler doesn't return `created`.  I can change that, though.

Reply:
Oh whatever then on that one. Still would be nice for `OwnAccount` and `ImportSecretKey` if they have easily compatible signatures.